### PR TITLE
Update __init__.py

### PR DESCRIPTION
**Pull request by henryotunuya**
Fixed a url error. 
Clicking the link takes you to "https://pypi.org/project/clintest." with the dot.

### CODE REVIEW

File: clintest/__init__.py
Comment (henryotunuya) [inline clintest/__init__.py:1]: @teiesti [#L146](https://github.com/henryotunuya/clintest/blob/6f9932ec85e38f89390b5dbe925b34557ad8df20/clintest/__init__.py#L146) is not really clear.

**Review by teiesti — DISMISSED** [review]

File: clintest/__init__.py
Comment (teiesti) [reply clintest/__init__.py:1]: Can you specify what you think is not clear?

**Comment by teiesti** [comment]
Thanks for your contribution. I will merge these changes. However, I am currently having problems with the automated tests. For some reason, they did not run for your pull request which is now blocking the merge.

File: clintest/__init__.py
Comment (henryotunuya) [reply clintest/__init__.py:1]: Insteads of:

> This is where `clintest.test.Record`, a test that can be wrapped around any other test, comes in handy.

This (or something which does not break the reading flow) reads better. 
> This is where `clintest.test.Record`$^1$ comes in handy. $^1$ note: a test that can be wrapped around any other test.

Comment (teiesti) [reply clintest/__init__.py:1]: Okay. Done.

**Pull request merged**